framework/resource_router.go

```diff
@@ -57,7 +57,7 @@ func (r *ResourceRouter) ResourceSet(obj interface{}) (*ResourceSet, error) {
 		if err != nil {
 			r.logger.Log("function", "ResourceSet", "level", "warning", "message", "cannot create accessor for object", "object", fmt.Sprintf("%#v", obj), "stack", fmt.Sprintf("%#v", err))
 		} else {
-			r.logger.Log("level", "debug", "message", "no resource set for reconciled object", "object", accessor.GetSelfLink())
+			r.logger.Log("function", "ResourceSet", "level", "debug", "message", "no resource set for reconciled object", "object", accessor.GetSelfLink())
```

Comment:
The other log above has the function key. Uncertain how useful this really is but just making this consistent now.